wallet/src/de/schildbach/wallet/ui/username/UsernameVotingInfoFragment.kt

```diff
@@ -0,0 +1,46 @@
+package de.schildbach.wallet.ui.username
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.activityViewModels
+import androidx.lifecycle.lifecycleScope
+import androidx.navigation.fragment.findNavController
+import dagger.hilt.android.AndroidEntryPoint
+import de.schildbach.wallet.ui.dashpay.DashPayViewModel
+import de.schildbach.wallet_test.R
+import de.schildbach.wallet_test.databinding.FragmentUsernameVotingInfoBinding
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.launch
+import org.dash.wallet.common.ui.viewBinding
+import org.dash.wallet.common.util.safeNavigate
+
+@AndroidEntryPoint
+@ExperimentalCoroutinesApi
```

Comment:
I don't think this is needed here.

Also, instead of `@ExperimentalCoroutinesApi` and similar annotations, let's use `@OptIn(ExperimentalCoroutinesApi::class)` (in places where it's needed). It has the benefit of not being propagated everywhere.

We have added a compiler option for this.